src/vs/editor/browser/services/codeEditorServiceImpl.ts

```diff
@@ -310,7 +310,7 @@ class DecorationRenderHelper {
 				cssTextArr.push(strings.format(this._CSS_MAP.contentIconPath, opts.contentIconPath.toString(true).replace(/'/g, '%27')));
```

Comment:
Migrated to separate PR. please see https://github.com/Microsoft/vscode/pull/18361/files